app/helpers/image-type.js

```diff
@@ -0,0 +1,8 @@
+import { helper } from '@ember/component/helper';
+
+export function imageType(params/*, hash*/) {
+  const type = params[0] || "";
+  return type.startsWith('image/');
+}
+
+export default helper(imageType);
```

Comment:
naming nitpick... something like `is-image-type` would give a better idea that this helper returns a boolean, rather than the image content type

Reply:
haha yup that was mildly annoying me too, will change